src/QMCWaveFunctions/RotatedSPOs.cpp

```diff
@@ -19,7 +19,13 @@
 namespace qmcplusplus
 {
 RotatedSPOs::RotatedSPOs(const std::string& my_name, std::unique_ptr<SPOSet>&& spos)
-    : SPOSet(my_name), OptimizableObject(my_name), Phi(std::move(spos)), nel_major_(0), params_supplied(false)
+    : SPOSet(my_name),
+      OptimizableObject(my_name),
+      Phi(std::move(spos)),
+      nel_major_(0),
+      params_supplied(false),
+      use_this_copy_to_apply_rotation_(false),
```

Comment:
Why not just initialize this in the class declaration? The reason to do it here would be if it was dependent on a constructor argument.

Reply:
Habit. Will change it.